functions.js

```diff
@@ -0,0 +1,37 @@
+const symbols = require('./symbols');
+
+module.exports = {
+	formatOptions(options) {
+		let formatted = '';
+		let keys = Object.keys(options);
+		for (let i = 0; i < keys.length; i++) {
+			formatted += `${keys[i]} ${options[keys[i]]}\n`;
+		}
```

Comment:
Me sitting here not sure if this is better than the map version....

I'm probably a bit to functionally trigger ready. I think this is fine